### DISCUSSION TITLE
Run lighthouse from selected servers

### DISCUSSION THREAD

**Discussion by pavanseniorassociate** [GoogleChrome/lighthouse#14502] [Q&A]
My website is for Germany and  I want to run lighthouse from German region/server or nearest regions/servers.
How can I do that ??

**Comment by connorjclark** [comment]
https://pagespeed.web.dev/ will run Lighthouse on the closest server near you (the user of PSI, not whatever location the website server resides) that that we have.

@exterkamp for how granular we can be about the locations here.

**Reply by pavanseniorassociate** [reply]
@connorjclark Thanks for your quick response.
I am trying to automate lighthouse for my application so could you please suggest me how can I do from command line??
Rest API is providing for light house.??If provided , can we run with nearest server???

Thanks & Regards 
Pavan Pothuganti

**Comment by pavanseniorassociate** [comment]
@connorjclark  ??
